dev/devicelab/README.md

```diff
@@ -26,10 +28,16 @@ To run a test, use option `-t` (`--task`):
 
 ```sh
 # from the .../flutter/dev/devicelab directory
-dart bin/run.dart -t {NAME_OF_TEST}
+dart bin/run.dart -t {NAME_OR_PATH_OF_TEST}
 ```
 
-You can see the test names in the `manifest.yaml` file in this directory or by looking in `bin/tasks/`. Do not include the `.dart` file extension. For example, `dart bin/run.dart -t complex_layout__start_up`.
+Where `NAME_OR_PATH_OF_TEST` can be either of:
+
+- the _name_ of a task, which you can find in the `manifest.yaml` file in this
+  directory. Example: `complex_layout__start_up`.
+- the path to a Dart _file_ corresponding to a task, which resides in `bin/tasks`.
+  Tip: most shells support path auto-completion using the Tab key. Example:
+  `bin/tasks/complex_layout__start_up.dart`.
```

Comment:
Task Dart files must go in `bin/tasks` and have a `.dart` extension. Names have no slashes or extensions.